proto/teleport/lib/teleterm/v1/target_health.proto

```diff
@@ -0,0 +1,32 @@
+/*
+ * Teleport
+ * Copyright (C) 2023  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+syntax = "proto3";
+
+package teleport.lib.teleterm.v1;
+
+option go_package = "github.com/gravitational/teleport/gen/proto/go/teleport/lib/teleterm/v1;teletermv1";
+
+// TargetHealth describes the health status of network connectivity between
+// an agent and a resource.
+message TargetHealth {
+  // status is the health status, one of "", "unknown", "healthy", "unhealthy".
+  string status = 1;
+  // error is the health check error observed.
+  string error = 2;
+}
```

Comment:
i also wasn't sure, if i should define own type like this, or [import from original](https://github.com/gravitational/teleport/blob/16aa45237578f4fb4aa7be253f7902082c13f0fb/api/proto/teleport/legacy/types/types.proto#L8354)?

i defined own type so we can get rid of the snake casing, but made me pause...

Reply:
If I remember correctly, importing from `legacy/` is problematic, and would require us to add generating JS code for it.
It's better to copy the message.